### ISSUE TITLE
Subclipse doesn't work on Ubuntu 20.04 (SVN 1.13.0)

### ISSUE THREAD

**Issue by waveywhite** [subclipse/subclipse#129]
Hi,

On upgrading to Ubuntu 20.04 LTS, I found that subclipse has stopped working. The issue is that Ubuntu 20.04 uses subversion 1.13 and libsvn-java 1.13, but subversion only supports up to 1.11.

Is there any timeline on support for 1.13? I can imagine this will quickly become a growing issue as peole switch to the latest Ubuntu LTS release.

Thanks
-David.

**Comment by markphip** [comment]
I will probably be upgrading to 20.04 next week so I cannot try this until then.  I through together an update site that should have the 1.13.0 JavaHL here:  https://dl.bintray.com/subclipse/releases/javahl/1.13.0/

I did something similar for 1.12 that worked so hopefully this will too.  I do not intend to officially release a 1.13 version and do not even have Windows binaries available for that version to do a release.  I am aiming at making 1.14.0 the one and only release I support going forward as that is the next and probably last Subversion LTS release.  Hopefully once it is out the distros will move to it sometime later.

**Comment by markphip** [comment]
See #130 

I think we can support SVN 1.12 thru 1.14 using a single version of our JavaHL plugin.  I have a new update site I am using for that. 

https://dl.bintray.com/subclipse/releases/javahl/1.14.x/

It would be great to have confirmation it works on Ubuntu with 1.13.  I will test it myself in a couple weeks when I upgrade. I have a colleague who went first and is having problems with 20.04 so I am waiting for him to work it out before I take the leap.

**Comment by markphip** [comment]
I was able to test this on OSX using Homebrew 1.13.0 version of SVN and the 1.14.x update site I posted.  Everything worked fine, so I would assume it will also work on Ubuntu 20.04.

**Comment by waveywhite** [comment]
Update - I've tested this on Ubuntu 20.04 and it works beautifully. Thank you.
Any chance you could change the description of the JavaHL plugin? It says it's only required on Windows, which is confusing.

**Comment by markphip** [comment]
I made a couple of cleanups of the text but nothing is jumping out at me that you might be referencing.  Unless you are referring to a README or Wiki page on this site.  But in the source code I am not sure I found anything. Do you have a screen shot?

**Comment by waveywhite** [comment]
It's the description in the details box:

![image](https://user-images.githubusercontent.com/2602241/80601472-c57dcb00-8a25-11ea-9edf-eb455a5e083f.png)

**Comment by markphip** [comment]
OK, I did fix that one. I did not think that one would show in the installation UI.

**Comment by hontvari** [comment]
I was able to install this version on Eclipse 2020-03, Ubuntu 20.04. First I installed Subclipse without JavaHL 1.10 and 1.11. Then I restarted Eclupse but I have to wait for several minutes, until I got the notification about the subversion-gnome-keyring issue. After that I installed JavaHL from the update site mentioned above:  https://dl.bintray.com/subclipse/releases/javahl/1.14.x/ .

Window/Preferences/Team/SVN/SVN interface/Client shows JavaHL (JNI) 1.13.0 (r1867053)

At first look Subclipse works.
